index.html

```diff
@@ -0,0 +1,383 @@
+          <a href="#"><img src="./assets/img/toggle.png" alt="toggle" /></a>
+        </div>
+      </nav>
+    </header>
+    <section class="showcase">
+      <div class="showcase_text">
+        <h1 id="tagline">
+          Biftu <br />
+          Girma
+        </h1>
+        <p>
+          Hello! I am a software developer! I can help you build a product,
+          feature or website. Take a look at my work. <br /><br />
+          If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+
+        <a class="primary-btn" href="#">Start collaboration</a>
+      </div>
+
+      <div class="showcase_img">
+        <img
+          src="./assets/img/headline_img.png"
+          alt="headline_img"
+          class="headline_img"
+        />
+      </div>
+    </section>
+
+    <section class="works">
+      <div class="work-card">
+        <h2 id="tagline">Projects</h2>
+        <img src="./assets/img/laptop.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
+          </div>
+          <div>
+            <a href="#" class="primary-btn-2"
+              >See this project
+              <img
+                src="./assets/img/right_arrow.png"
+                alt="right_arrow"
+                class="right_arrow"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="work-card">
+        <img src="./assets/img/laptop 2.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
+          </div>
+          <div>
+            <a href="#" class="primary-btn-2"
+              >See this project
+              <img
+                src="./assets/img/right_arrow.png"
+                alt="right_arrow"
+                class="right_arrow"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+      <div class="work-card">
+        <img src="./assets/img/laptop 3.png" alt="laptop" />
+        <div class="work-card-text">
+          <h3>Project name goes here</h3>
+          <div class="lang-container">
+            <div>
+              <h4 id="lang">HTML/CSS</h4>
+            </div>
+            <div>
+              <h4 id="lang">Ruby on Rails</h4>
+            </div>
+            <div>
+              <h4 id="lang">JavaScript</h4>
+            </div>
```

Comment:
- [ ] Kindly wrap the technologies in `ul` tag 👍 and each of them a `li`. This way will be semantically correct and the screen readers